services/web/server/tests/unit/with_dbs/03/resource_usage/test_pricing_plans.py

```diff
@@ -0,0 +1,83 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-statements
+
+
+from datetime import datetime, timezone
+from decimal import Decimal
+
+import pytest
+from aiohttp import web
+from aiohttp.test_utils import TestClient
+from models_library.api_schemas_resource_usage_tracker.pricing_plans import (
+    PricingUnitGet,
+    ServicePricingPlanGet,
+)
+from models_library.resource_tracker import PricingPlanClassification
+from pytest_mock.plugin import MockerFixture
+from pytest_simcore.helpers.utils_assert import assert_status
+from pytest_simcore.helpers.utils_login import UserInfoDict
+from simcore_service_webserver.db.models import UserRole
+
+_PRICING_UNIT_GET: PricingUnitGet = PricingUnitGet(
+    pricing_unit_id=1,
+    unit_name="SMALL",
+    current_cost_per_unit=Decimal(5),
+    current_cost_per_unit_id=1,
+    default=True,
+    specific_info={},
+)
+
+_PRICING_PLAN_GET: ServicePricingPlanGet = ServicePricingPlanGet(
+    pricing_plan_id=1,
+    display_name="Sleeper pricing plan",
+    description="",
+    classification=PricingPlanClassification.TIER,
+    created_at=datetime.now(tz=timezone.utc),
+    pricing_plan_key="sleeper-pricing-plan",
+    pricing_units=[_PRICING_UNIT_GET],
+)
+
+
+@pytest.fixture
+def mock_resource_tracker_client(mocker: MockerFixture) -> tuple:
```

Comment:
For interaction with backend services, we should instead start to mock the responses of the service at transport level. Moreover, this opens you to the possibility of testing failures (e.g. 500s etc) coming from the RUT service

You can emulate this example from the catalog
https://github.com/ITISFoundation/osparc-simcore/blob/54c8647744d09e9775268dc23679c1ece6ec15da/services/web/server/tests/unit/with_dbs/01/test_catalog_api.py#L25-L45

Reply:
done, thanks!